src/Authentication/Authentication/Utilities/DependencyAssemblyResolver.cs

```diff
@@ -94,35 +116,4 @@ private static Assembly ResolvingHandler(object sender, ResolveEventArgs args)
             return null;
         }
     }
-
-    internal class AssemblyLoadContextProxy
```

Comment:
Why are you removing the AssemblyLoadContext proxy class? The class is needed to isolate the module's ALC to avoid assembly conflicts with other modules. See https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/1867 and https://learn.microsoft.com/en-us/powershell/scripting/dev-cross-plat/resolving-dependency-conflicts?view=powershell-7.3